### PR TITLE
Add kewl status badge to Readme

### PR DESCRIPTION
**Pull request by prateekgogia**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+![Build Status](https://img.shields.io/github/workflow/status/awslabs/karpenter/Presubmit)
+![GitHub stars](https://img.shields.io/github/stars/awslabs/karpenter)
```
Comment (bwagner5) [inline README.md:2]: do we need forks and stars since they're at the top right of the screen anyways?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+![GitHub forks](https://img.shields.io/github/forks/awslabs/karpenter)
+![GitHub License](https://img.shields.io/github/license/awslabs/karpenter)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/awslabs/karpenter/issues)
+
```
Comment (bwagner5) [inline README.md:6]: goreportcard would be nice too:

`https://goreportcard.com/badge/github.com/awslabs/karpenter`

**Review by njtran — COMMENTED** [review]
LGTM!

Let me know what you think about the following?
[forthebadge](https://forthebadge.com)
`[forthebadge](https://forthebadge.com)`
[forthebadge](https://forthebadge.com)
`[forthebadge](https://forthebadge.com)`

File: README.md (hunk 2 above)
Comment (JacobGabrielson) [reply README.md:6]: would also be nice to have code coverage, although that might require more changes to the build system?

File: README.md (hunk 1 above)
Comment (prateekgogia) [reply README.md:2]: Top level stars don't show up on forks, so if I am reading a fork some good to know information may be?

File: README.md (hunk 2 above)
Comment (prateekgogia) [reply README.md:6]: nice will add this one.

Comment (prateekgogia) [reply README.md:7]: > would also be nice to have code coverage, although that might require more changes to the build system?

yeah, will require to integrate with codecov or something similar, will need to spend sometime comparing whats the right way to do it.

**Review by ellistarn — APPROVED** [review]
nice!

**Pull request merged**